apps/nextra/pages/en/build/create-aptos-dapp/templates/token-staking-dapp.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: "Create Aptos Dapp Token Staking Dapp Template"
```

Comment:
```suggestion
title: "create-aptos-dapp Token Staking Template"

```

Reply:
hmm all other template pages use this title format